app/views/admin/users/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <h2>Members</h2>
 
-<p>There are currently <%= User.confirmed.count %> members</p>
```

Comment:
what's this change for?

Reply:
Ah, crud thought I'd removed that, it's a pullover from the unconfirmed page, the wording should change not the variable used